README.md

```diff
@@ -393,6 +382,69 @@ export const $newPosts = computed([$lastVisit, $posts], (lastVisit, posts) => {
 })
 ```
 
+### Effects
+
+In spite of having ability to use atom's subscribe/notify methods for subscribing on changes, it might be a bit tricky to track changes at multiple atoms at once. `effect` is designed to solve this problem!
+
+`effect` runs its callback on the start, with initial values, as well as on any store(s) change. If callback returns cleanup function it will be performed before next callback run. Besides that, `effect` returns own cleanup function, which allows cancelling the whole effect.
+
+`effect` can be used as analogue of `subscribe` method on one store:
+
+```ts
+import { atom, effect } from 'nanostores'
+
+const $num = atom(10)
+
+effect($atom, num => {
+  console.log(`Number is ${num}`)
+})
```

Comment:
We will need multi storage subscriber example (and link from the first mention of `listen` to answer the question how to listen multiple stores).

Reply:
I mentioned `listen` as a way to subscribe on changes, not as technically the same method (since it doesn't run callback on initialization).
If you think it might confuse users, let's just remove this mention, because otherwise example will require separate counter/boolean flag to exclude initial run. Like:

```js
const $atom1 = atom(1);
const $atom2 = atom(2);

let isFirstRun = true;
effect([$atom1, $atom2], (atom1, atom2) => {
  if (isFirstRun) {
    isFirstRun = false
    return
  }

  console.log(atom1 + atom2);
});
```

We could add third parameter, to the `effect` to have control over this, but I believe it is rare case, and it can be easily handled in above-mentioned way. I prefer to not overcomplicate API (I don't remember such option in other state managers either, btw)